src/contexts/MapContext.js

```diff
@@ -14,13 +17,27 @@ const MapContextProvider = (props) => {
     const init = async function () {
       if (mapView) {
         await initView(mapView);
-        setSelection(dummyFeature);
+
+        if (match.params.fid) {
+          console.log('querying for:', match.params.fid);
+
+          const feature = await getTrailFeature(match.params.fid);
+          console.log(feature);
+          setSelection(feature);
+        }
+
         setReady(true);
       }
     };
     init();
   }, [mapView]);
```

Comment:
ESLint complains here that:

![image](https://user-images.githubusercontent.com/3399064/87491456-67fd2180-c605-11ea-8617-0296a09fa9e8.png)

... but adding `match` to the dependency array causes an infinite loop of maps getting created 😅 

Any ideas @odoe or @michaelgaigg?